Sources/Paywalls/Components/Common/PaywallComponentPropertyTypes.swift

```diff
@@ -109,7 +109,7 @@ public extension PaywallComponent {
 
     }
 
-    enum WidthSizeType: String, Codable, Sendable, Hashable, Equatable {
+    @frozen enum WidthSizeType: String, Codable, Sendable, Hashable, Equatable {
         case fit, fill, fixed
```

Comment:
Should we allow something like `fixedPercentage`?

Reply:
There has been A LOT of discussion about this 😅 We decided to leave this out (for now) but will probably introduce some type of "grid" component that has percentage like this in a more structured way